packages/bsky/tests/views/author-feed.test.ts

```diff
@@ -165,12 +165,14 @@ describe('pds author feed views', () => {
         },
       )
 
-    const { data: postBlock } = await agent.api.app.bsky.feed.getAuthorFeed(
-      { actor: alice },
-      { headers: await network.serviceHeaders(carol) },
-    )
-
-    expect(postBlock.feed.length).toEqual(0)
+    try {
+      const { data: postBlock } = await agent.api.app.bsky.feed.getAuthorFeed(
```

Comment:
this test actually wont fail if it doesn't throw

normally use the following syntax
```ts
await expect(function).rejects.toThrow("Profile not found")
```